jobs/tcp_router/spec

```diff
@@ -86,6 +90,19 @@ properties:
     description: "Fail the tcp router if routing_api.reserved_system_component_ports conflict with ports in existing router groups."
     default: "false"
 
+  tcp_router.drain_wait:
+    description: |
+      Delay in seconds after shut down is initiated before haproxy stops listening.
+      During this time haproxy will reject requests to the /health endpoint.
+      This accommodates requests forwarded by a load balancer until it considers the tcp_router unhealthy.
+    default: 20
+  tcp_router.load_balancer_healthy_threshold:
```

Comment:
"threshold" seems like a weird word for this to me.... but now I see that it matches almost identically to what we have for [gorouter](https://github.com/cloudfoundry/routing-release/blob/3c7e25fe643a905bb4b3b2edee25a23fcd412132/jobs/gorouter/spec#L98-L100). So I guess I approve of consistency.